### PR TITLE
Update main.js

### PR DESCRIPTION
**Pull request by Yay295**
Made some variables const. It doesn't work in IE<11, but they can't play webm either.
Changed some jQuery to Vanilla Javascript.
Removed console.log(volume changed) spam.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -57,8 +57,8 @@ function hideMenu() {
 
 // Play/Pause Button
 function playPause() {
-  // Set media player variable.
-  var video = $('#bgvid')[0];
+  // Set media player constant.
+  const video = document.getElementById("bgvid");
```
Comment (TheHowl) [inline js/main.js]: I understand the const part, but why change jquery to DOM here and leave all the rest (except few cases) jquery?

File: js/main.js [hunk 2]
```diff
@@ -202,14 +194,14 @@ function konamicheck(k)
...
-    $('#skip-right').toggleClass('fa-spin');
-    $('#pause-button').toggleClass('fa-spin');
+    
+    document.getElementById("menubutton").toggleClass('fa-spin');
+    document.getElementById("bgvid").toggleClass('fa-spin');
+    document.getElementById("getnewvideo").toggleClass('fa-spin');
+    document.getElementById("autonext").toggleClass('fa-spin');
+    document.getElementById("skip-left").toggleClass('fa-spin');
+    document.getElementById("skip-right").toggleClass('fa-spin');
+    document.getElementById("pause-button").toggleClass('fa-spin');
```
Comment (TheHowl) [inline js/main.js]: just one, big question:

# WHY

Also, in vanilla javascript there's no .toggleClass - http://stackoverflow.com/q/18880890

**Comment by TheHowl** [comment]
You've made the same mistake as the last comment in multiple parts. Check what's the vanillajs equivalent before changing jquery to DOM. Also, I don't get what's the fucking point in changing all that stuff to vanillajs? I mean, if you really wish, you can erase completely jquery from animeopenings converting all the stuff to vanillajs, but that would be useless IMHO as jquery improves readability of code and a lot of the javascript is jquery-dependent (check the Konami plugin).

**Comment by Yay295** [comment]
"document.getElementById" is a lot more descriptive than "$('#".
(It's also a whole lot faster, but our code wasn't executing slowly before so that's not as big an issue.)

Let's just go with the const changes though.

**Pull request closed without merging**